jekyll/_cci2/runner-overview.adoc

```diff
@@ -0,0 +1,106 @@
+
+NOTE: The CircleCI runner is currently only available if you are on the Scale Plan. Please reach out to your sales representative for information on how to sign up for the Scale Plan.
+
+toc::[]
+
+== Introduction
+
+CircleCI runner enables you to use your own infrastructure for running jobs. This means you will be able to build and test on a wider variety of architectures and environments, in addition to providing you with additional control over the security of the environment. The diagram below illustrates how CircleCI Runner can be implemented in an environment.
+
+.Runner Architecture
+image::runner-overview-diagram.png[CircleCI Runner Architecture]
+
+== Runner use cases
+
+There are two key use cases CircleCI aims to meet with the runner:
+
+* Security & compliance requirements - CircleCI understands that some customers require running jobs on on-premises or limited-access infrastructure due to strict security requirements. Some things that the runner enables are:
+** IP restrictions - Runners can have static IP addresses that you can control.
+** Identity Access Management (IAM) permissions - If you set up runners in AWS, they can be assigned IAM permissions.
+** Monitor the operating system.
+** Connect to private networks.
+
+* Unique compute requirements - Customers who need to run jobs on an environment or architecture that CircleCI does not offer as a resource class can use the runner to fill that need.
+
+== Runner operation
+
+Once CircleCI runner is installed, the runner polls circleci.com for work, runs jobs, and returns status, logs, and artifacts to CircleCI. When the runner is not running a job, it will auto-update itself when a new version is released.
+
+The runner consists of two components: the Launch Agent and the Task Agent.
+
+* Launch Agent (launch-agent) - manages gathering the information required to run a task (defined as a parallel run of a job) while also downloading and launching a Task Agent process.
+* Task Agent (task-agent) - handles running a task retrieved and configured by the Launch Agent.
+
+The system has been designed to allow administrators to configure the task-agent to run with a lower level of privileges than the launch-agent. Any user who is able to execute a job will be able to gain the same privileges as task-agent. The instructions below are the recommended deployment which follows this approach (Launch Agent will run as root, but Task Agent will run as circleci).
+
+== CircleCI runner limitations
+
+There are a few limitations you should be aware of when using CircleCI runner:
+
+Currently, the runner officially supports the following platforms:
+
+* Intel + Ubuntu
+* Arm64 + Ubuntu
+* Intel + macOS
+
+Additional platforms have been tested but are not yet officially supported by the runner. For more information on platforms that have been tested, but are not supported, and platforms that have not been tested, please refer to the Runner Installation [add link here when ready] page.
```

Comment:
need to manage this interlinking - maybe merge everything into a runner feature branch together and sort links before deploying to production?